regression/contracts/assigns_enforce_01/main.c

```diff
@@ -0,0 +1,18 @@
+#include <assert.h>
+
+int z;
+
+int foo(int *x) __CPROVER_assigns(z, *x)
```

Comment:
Why is `z` in the assigns clause?

Reply:
The intent was to show that the assigns clause should be interpreted as "may assign" rather than "assigns exactly"; it is okay to put things in the assigns clause which are not assigned without issue. I have added a comment in this regression test to make this more clear.